### ISSUE TITLE
adversarial_training_FBF.py example Error. 

### ISSUE THREAD

**Issue by tylershumaker** [Trusted-AI/adversarial-robustness-toolbox#2014]
**Describe the bug**
Runtime error occurred when running examples/adversarial_training_FBF.py.

```
Adversarial Training FBF - Epochs:   0%|          | 0/30 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "adversarial_training_FBF.py", line 219, in <module>
    trainer.fit_generator(art_datagen, nb_epochs=30)
  File "/opt/conda/lib/python3.8/site-packages/art/defences/trainer/adversarial_trainer_fbf_pytorch.py", line 181, in fit_generator
    _train_loss, _train_acc, _train_n = self._batch_process(x_batch, y_batch, l_r)
  File "/opt/conda/lib/python3.8/site-packages/art/defences/trainer/adversarial_trainer_fbf_pytorch.py", line 214, in _batch_process
    delta_grad = self._classifier.loss_gradient(x_batch + delta, y_batch)
  File "/opt/conda/lib/python3.8/site-packages/art/estimators/classification/pytorch.py", line 834, in loss_gradient
    y_preprocessed = self.reduce_labels(y_preprocessed)
  File "/opt/conda/lib/python3.8/site-packages/art/estimators/classification/pytorch.py", line 275, in reduce_labels
    is_one_hot = torch.equal(y.floor(), y.ceil())
RuntimeError: "floor_cuda" not implemented for 'Long'
```

The code causing the error was added here: [https://github.com/Trusted-AI/adversarial-robustness-toolbox/commit/3f7899593238aa38522c21e77f1dcc76afa3ce0b](https://github.com/Trusted-AI/adversarial-robustness-toolbox/commit/3f7899593238aa38522c21e77f1dcc76afa3ce0b)

Import needed updated as well:
from art.classifiers import PyTorchClassifier
to
from art.estimators.classification import PyTorchClassifier

**To Reproduce**
1. Run examples/adversarial_training_FBF.py
2. See error

**Expected behavior**
Example to complete successfully. 

**System information (please complete the following information):**
 - OS: Ubuntu 18.04.6 LTS
 - Python version 3.8
 - ART version 1.13.0
 - Pytorch

**Comment by beat-buesser** [comment]
Hi @tylershumaker Thank you very much for opening this issue!  I have been able to reproduce the import. I'm still working on the RuntimeError. Which version of PyTorch are you using and are you running on GPU?

**Comment by beat-buesser** [comment]
I have been able to reproduce this issue. we'll fix it with ART 1.13.1.

@tylershumaker Thank you very much for reporting this issue!
